### ISSUE TITLE
Do we need to subtract mean pixel values when extracting cnn feautre

### ISSUE THREAD

**Issue by danieljf24** [apache/incubator-mxnet#7469]
Hi! I followed this [notebook](https://github.com/dmlc/mxnet-notebooks/blob/master/python/how_to/predict.ipynb) to extract image feature using pre-trained CNN model. I wonder do we need to subtract mean pixel values of imagenet before feature extraction. Other deep learning frameworks usually have this process, but I didn't find it in this notebook.

**Comment by edmBernard** [comment]
The model in your notebook use batch normalization, in this case "subtract mean pixel values" is useless.
https://arxiv.org/abs/1502.03167

**Comment by chinakook** [comment]
If the model have used batch normalization following the data layer(data batch normalization), you can ignore the mean pixel values and additional std scale sometimes. However, if the batch normalization have used after some layers except data layer, you should subtract the mean values and multiply a std scale sometimes.

**Comment by szha** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to ping me to reopen if this is still an active issue. Thanks!
Also, do please check out our [forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)) for general "how-to" questions.
